superset-frontend/cypress-base/cypress/integration/dashboard/nativeFilters.test.ts

```diff
@@ -288,6 +286,51 @@ describe('Nativefilters Sanity test', () => {
       'By default, each filter loads at most 1000 choices at the initial page load. Check this box if you have more than 1000 filter values and want to enable dynamically searching that loads filter values as users type (may add stress to your database).',
     );
   });
+  it("User can check 'Filter has default value'", () => {
+    cy.get(nativeFilters.filterFromDashboardView.expand).click({ force: true });
+    cy.get(nativeFilters.createFilterButton)
+      .should('be.visible')
+      .click({ force: true });
+    cy.get(nativeFilters.modal.container).should('be.visible');
+
+    cy.get(nativeFilters.modal.container)
+      .find(nativeFilters.filtersPanel.datasetName)
+      .click()
+      .type('wb_health_population{enter}');
+    cy.get(nativeFilters.modal.container)
+      .find(nativeFilters.filtersPanel.filterName)
+      .click()
+      .type('country_name');
+    cy.get('.loading inline-centered css-101mkpk').should('not.exist');
```

Comment:
I think this might need to be like the following instead? Also, I would avoid pointing to generated classes like this one if possible `css-101mkpk`

```suggestion
    cy.get('.loading .inline-centered .css-101mkpk').should('not.exist');
```

Reply:
I also think it is not necessary to check the generated classes, remove that line